clients/gtest/blas1_gtest.yaml

```diff
@@ -21,70 +25,193 @@ Definitions:
+      - dot:   *double_precision_complex_real
+      - dotc:  *double_precision_complex_real
+      - scal:  *single_double_precisions_complex_real
+      - scal:  *single_double_complex_real_in_complex_out
+      - swap:  *single_double_precisions_complex_real
+      - rot:   *rot_precisions
+      - rotm:   *single_double_precisions_complex_real
+
+  - name: blas1_batched
+    category: pre_checkin
+    N: [ 1048576 ]
+    incx_incy: *incx_incy_range
+    alpha_beta: *alpha_beta_range
+    alphai_betai: *alphai_betai_range
+    batch_count: [2]
+    function:
+      - scal_batched: *single_double_precisions_complex_real
+      - scal_batched:  *single_double_complex_real_in_complex_out
+
+  - name: blas1_strided_batched
+    category: pre_checkin
+    N: [ 1048576 ]
+    incx_incy: *incx_incy_range
+    alpha_beta: *alpha_beta_range
+    alphai_betai: *alphai_betai_range
+    batch_count: [2]
+    strideScale: [ 1.5 ]
+    function:
+      - scal_strided_batched: *single_double_precisions_complex_real
+      - scal_strided_batched:  *single_double_complex_real_in_complex_out
+
+  - name: blas1
+    category: nightly
+    N: [ 4000000 ]
+    incx_incy: *incx_incy_range
+    alpha_beta: *alpha_beta_range
+    alphai_betai: *alphai_betai_range
+    function:
+      - nrm2:  *double_precision_complex_real
+      - asum:  *double_precision_complex_real
+      - iamax: *single_double_precisions_complex_real
+  #   - iamin: *single_double_precisions_complex_real # broken for now -- cause unknown
+      - axpy:  *half_single_precisions_complex_real
+      - copy:  *single_double_precisions_complex_real
+      - dot:   *double_precision_complex_real
+      - dotc:  *double_precision_complex_real
+      - scal:  *single_double_precisions_complex_real
+      - scal:  *single_double_complex_real_in_complex_out
+      - swap:  *single_double_precisions_complex_real
+      - rot:   *rot_precisions
+      - rotm:   *single_double_precisions_complex_real
+
+  - name: blas1_batched
+    category: nightly
+    N: [ 4000000 ]
+    incx_incy: *incx_incy_range_small
+    alpha_beta: *alpha_beta_range
+    alphai_betai: *alphai_betai_range
+    batch_count: [ 2 ]
+    function:
+      - scal_batched: *single_double_precisions_complex_real
+      - scal_batched:  *single_double_complex_real_in_complex_out
+
+  - name: blas1_batched
+    category: nightly
+    N: [ 10 ]
+    incx_incy: *incx_incy_range_small
+    alpha_beta: *alpha_beta_range
+    alphai_betai: *alphai_betai_range
+    batch_count: [ 5000 ]
+    function:
+      - scal_batched: *single_double_precisions_complex_real
+      - scal_batched:  *single_double_complex_real_in_complex_out
+
+  - name: blas1_strided_batched
+    category: nightly
+    N: [ 4000000 ]
+    incx_incy: *incx_incy_range_small
+    alpha_beta: *alpha_beta_range
+    alphai_betai: *alphai_betai_range
+    batch_count: [ 2 ]
+    strideScale: [ 1.2 ]
+    function:
+      - scal_batched: *single_double_precisions_complex_real
```

Comment:
I think this is meant to be strided_batched?

Reply:
Good catch, fixed in e906818.